site/content/docs/browser-checks/_index.md

```diff
@@ -25,135 +26,121 @@ Your critical interactions might be:
-await page.click('.btn.btn-success.btn-block')
+  await page.locator('input[type="email"]').type('john@example.com') // 4
+  await page.locator('input[type="password"]').type('mypassword') // 5
+  await page.getByRole('button', { name: 'Log In' }).click() // 5
 
-await page.waitForSelector('.status-table') // 4
-await page.screenshot({ path: 'checkly_dashboard.png' })
-await browser.close()
+  await page.getByTestId('home-dashboard-table').isVisible() // 6
+})
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
-**1. Initial declarations:** We first import a framework (Puppeteer or Playwright) to control a browser.
-We also declare a `browser` and a `page` variable.
+**1. Initial declarations:** We first import a framework (Playwright Test Runner or Playwright) to control the browser.
 
-**2. Initial navigation:** We use the `page.goto()` method to load the first page.
+**2. Establish environment:** We use the `page` fixture without having to initialise a browser and create a new page manually. See the documentation on [Fixtures](https://playwright.dev/docs/api/class-fixtures) to learn more.
 
-**3. Fill out input fields and submit:** Using the `page.type()` and `page.click()` methods we enter our email address and
-password and click the login button. You would normally use environment variables here to keep sensitive data
+**3. Initial navigation:** We use the `page.goto()` method to load the first page.
+
+**4. Fill out input fields and submit:** Using the `page.type()` method we enter our email address and
+password. You would normally use environment variables here to keep sensitive data
 out of your scripts. See [Login scenarios and secrets](/docs/browser-checks/login-and-secrets/) for more info.
 
-**4. Wait for dashboard and close:** The expected behaviour is that the dashboard loads. In our case we can recognise this
-by the correct loading of the `.status-table` element in the page. For a visual reference, we also take a screenshot.
-We then close the browser. Our check is done.
+**5. Click Login button:** We use Playwright's `getByRole()` locator to find the login button and also `.click()` on it right away.
+
+**6. Wait for the dashboard:** The expected behaviour is that the dashboard loads. We assess this by checking whether the element with the test ID `home-dashboard-table` is visible. The `getByTestId()` method is looking for elements where the `data-testid` attribute matches the provided value. Playwright Test will automatically tear down the `page` fixture after the test has finished.
 
 ## How do I create a Browser check?
 
-Every valid **[Puppeteer](https://github.com/GoogleChrome/puppeteer)**
-or **[Playwright](https://github.com/microsoft/playwright)** script is a valid Browser check. You can create these
-scripts in two ways:
+Every valid **[Playwright Test](https://playwright.dev/docs/intro)** or **[Playwright](https://github.com/microsoft/playwright)** script is a valid Browser check. You can create these scripts in two ways:
```

Comment:
@miliberlin this is really a question: is it true _every_ valid script in those is valid on Checkly? This is a strong statement and I was under the impression there were certain configs etc that we didn't really support.

Reply:
- As of now (i know it'll be fixed soon), PWT scripts that aren't requiring `@playwright/test` in the entry file, won't work and throw.

Also I could think of screenshot comparisons which are theoretically valid but won't work in Checkly.

Reply:
I rephrased this (see [last commit](https://github.com/checkly/checklyhq.com/pull/625/commits/f9fbe2e8a50ae4c91d1084af97c74115306a2065)) to make it less strong and add a link to the list of supported features. That list will probably be moved to the quickstart page once we get rid of the separate PWT page.